lib/matplotlib/tests/test_font_manager.py

```diff
@@ -111,12 +110,7 @@ def test_utf16m_sfnt():
 def test_find_ttc():
     fp = FontProperties(family=["WenQuanYi Zen Hei"])
     if Path(findfont(fp)).name != "wqy-zenhei.ttc":
-        if not os.environ.get("TRAVIS") or sys.platform != "linux":
-            pytest.skip("Font may be missing")
-        # Travis appears to fail to pick up the ttc file sometimes.  Try to
-        # rebuild the cache and try again.
-        fm._rebuild()
-        assert Path(findfont(fp)).name == "wqy-zenhei.ttc"
+        pytest.skip("Font may be missing")
```

Comment:
Are we sure, we can drop skipping on `sys.platform != "linux"`?

Reply:
We only needed the second half of that `or` to allow the skip to happen on the mac on travis.  I think this is OK, but I'm going to push a commit to this branch to install zenhei on azure for linux and OSX.